pkgs/top-level/python-packages.nix

```diff
@@ -6861,6 +6861,27 @@ in modules // {
     };
   };
 
+  grip = buildPythonPackage rec {
+    version = "4.3.2";
+    name = "grip-${version}";
+
+    src = pkgs.fetchFromGitHub {
```

Comment:
PyPi is typically preferred
https://pypi.org/project/grip/

Unless the tests aren't included in the archive but are in the repo.